.github/workflows/matlab.yml

```
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: MATLAB
+
+on:
+  push:
+    paths:
+      - '.github/workflows/matlab.yml'
+      - 'ci/scripts/matlab*.sh'
+      - 'matlab/**'
+      - 'cpp/src/arrow/**'
+  pull_request:
+    paths:
+      - '.github/workflows/matlab.yml'
+      - 'ci/scripts/matlab*.sh'
+      - 'matlab/**'
+      - 'cpp/src/arrow/**'
+
+concurrency:
+  group: ${{ github.repository }}-${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+
+  matlab:
+    name: MATLAB 
```

Could you use `${ARCHITECTURE} ${PLATFORM} ${LANGUAGE} ${NOTE}` format like other jobs? (See also other `.github/workflows/*.yml`.)

e.g.:

```suggestion
    name: AMD64 Ubuntu 20.04 MATLAB 
```

---

Thanks for the suggestion - this makes sense.